src/modules/Nfs.ycp

```diff
@@ -352,6 +353,60 @@
 	return true;
     }
 
+    /*
+     * Supports(a, b) is true if a must be mounted before b.
+     * That is, "/" Supports "/any", "/foo" Supports "/foo/bar",
+     * but "/foo" does not Support "/foof"
+     */
+    boolean Supports(any entry_a, any entry_b) {
```

Comment:
I am not sure if supports is good name, maybe depends_on is better ( with inverse logic )